validator/error.go

```diff
@@ -0,0 +1,13 @@
+package validator
+
+import (
+	"github.com/juju/errgo"
+)
+
+var (
+	noUnitsInGroupError     = errgo.New("no units in group")
+	badUnitPrefixError      = errgo.New("unit does not have group prefix")
+	mixedSliceInstanceError = errgo.New("group mixing scalable and non-scalable units")
+
+	groupsArePrefixError = errgo.New("group is prefix of another group")
```

Comment:
We have the convention to create errors and their assertion methods. So the assertion methods are missing here.